packages/ember-application/lib/system/application.js

```diff
@@ -1003,6 +1004,7 @@ Application.reopenClass({
 
     registry.injection('view', 'renderer', 'renderer:-dom');
     registry.register('view:select', SelectView);
+    registry.register('view:outlet', OutletView);
```

Comment:
If your position is that this is actually a public API (for example if folks are supposed to put a file at `app/views/outlet.js` that exports a custom OutletView), I suppose that is a different argument, but I do not think that is what you are suggesting...